### ISSUE TITLE
Failed to load file in public folder

### ISSUE THREAD

**Issue by keyding** [wxt-dev/wxt#579]
### Describe the bug

After modifying `srcDir`, the runtime reports an error:

```bash
WXT 0.17.10                                                                                                                                                                       8:54:17 PM
✔ Started dev server @ http://localhost:3000                                                                                                                                     8:54:18 PM
ℹ Pre-rendering chrome-mv3 for development with Vite 5.2.7                                                                                                                       8:54:18 PM
✔ Built extension in 659 ms                                                                                                                                                      8:54:18 PM
  ├─ .output/chrome-mv3/manifest.json               875 B
  ├─ .output/chrome-mv3/popup.html                  552 B
  ├─ .output/chrome-mv3/background.js               126.87 kB
  ├─ .output/chrome-mv3/chunks/popup-Cx96etly.js    7.41 kB
  ├─ .output/chrome-mv3/content-scripts/content.js  136.01 kB
  ├─ .output/chrome-mv3/icon/128.png                3.07 kB
  ├─ .output/chrome-mv3/icon/16.png                 559 B
  ├─ .output/chrome-mv3/icon/32.png                 916 B
  ├─ .output/chrome-mv3/icon/48.png                 1.33 kB
  ├─ .output/chrome-mv3/icon/96.png                 2.37 kB
  └─ .output/chrome-mv3/wxt.svg                     1.07 kB
Σ Total size: 281.03 kB
✔ Opened browser in 1.144 s                                                                                                                                                      8:54:20 PM
8:54:23 PM [vite] Internal server error: Failed to resolve import "/wxt.svg" from "src/entrypoints/popup/App.vue". Does the file exist?
  Plugin: vite:import-analysis
  File: /Users/caven/workspaces/_labs/wxt-public-issue/src/entrypoints/popup/App.vue:8:25
  11 |  });
  12 |  import { createElementVNode as _createElementVNode, createVNode as _createVNode, Fragment as _Fragment, openBlock as _openBlock, createElementBlock as _createElementBlock, pushScopeId as _pushScopeId, popScopeId as _popScopeId } from "vue";
  13 |  import _imports_0 from "/wxt.svg";
     |                          ^
  14 |  import _imports_1 from "@/assets/vue.svg";
  15 |  const _withScopeId = (n) => (_pushScopeId("data-v-89a223eb"), n = n(), _popScopeId(), n);
      at formatError (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:50796:46)
      at TransformContext.error (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:50790:19)
      at normalizeUrl (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:65935:33)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:66090:47
      at async Promise.all (index 3)
      at async TransformContext.transform (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:66011:13)
      at async Object.transform (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:51105:30)
      at async loadAndTransform (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:53858:29)
8:54:23 PM [vite] Pre-transform error: Failed to resolve import "/wxt.svg" from "src/entrypoints/popup/App.vue". Does the file exist?
9:06:35 PM [vite] Internal server error: Failed to resolve import "/wxt.svg" from "src/entrypoints/popup/App.vue". Does the file exist?
  Plugin: vite:import-analysis
  File: /Users/caven/workspaces/_labs/wxt-public-issue/src/entrypoints/popup/App.vue:8:25
  11 |  });
  12 |  import { createElementVNode as _createElementVNode, createVNode as _createVNode, Fragment as _Fragment, openBlock as _openBlock, createElementBlock as _createElementBlock, pushScopeId as _pushScopeId, popScopeId as _popScopeId } from "vue";
  13 |  import _imports_0 from "/wxt.svg";
     |                          ^
  14 |  import _imports_1 from "@/assets/vue.svg";
  15 |  const _withScopeId = (n) => (_pushScopeId("data-v-89a223eb"), n = n(), _popScopeId(), n);
      at formatError (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:50796:46)
      at TransformContext.error (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:50790:19)
      at normalizeUrl (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:65935:33)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:66090:47
      at async Promise.all (index 3)
      at async TransformContext.transform (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:66011:13)
      at async Object.transform (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:51105:30)
      at async loadAndTransform (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:53858:29)
      at async viteTransformMiddleware (file:///Users/caven/workspaces/_labs/wxt-public-issue/node_modules/vite/dist/node/chunks/dep-C-KAszbv.js:63631:32)
```

### To Reproduce

https://github.com/keyding/wxt-public-issue

Steps to reproduce the bug using the reproduction:

1. Bootstrap project: `bunx wxt@latest init wxt-public-issue`
2. Install dependencies: `bun install`
3. Create `src` folder
4. Moving the `assets`, `components`, `entrypoints` and  `public` to the `src` folder
5. Configuration file `wxt.config.ts` add `srcDir: 'src'`
6. Start dev mode: `bun run dev`
7. Click on the extension icon

### Expected behavior

Files in the `public` folder are correctly recognized.

### Screenshots

![CleanShot wxt-public-issues](https://github.com/wxt-dev/wxt/assets/20431351/7322a4a4-283c-455a-ad33-befca597a8a2)

### Environment

```
System:
  OS: macOS 14.4.1
  CPU: (10) arm64 Apple M1 Pro
  Memory: 47.94 MB / 16.00 GB
  Shell: 5.9 - /bin/zsh
Binaries:
  Node: 20.11.1 - ~/.volta/tools/image/node/20.11.1/bin/node
  Yarn: 1.22.19 - ~/.volta/bin/yarn
  npm: 10.2.4 - ~/.volta/tools/image/node/20.11.1/bin/npm
  pnpm: 8.15.1 - ~/.volta/bin/pnpm
  bun: 1.0.30 - ~/.bun/bin/bun
Browsers:
  Brave Browser: 123.1.64.109
  Chrome: 123.0.6312.87
  Safari: 17.4.1
npmPackages:
  wxt: ^0.17.0 => 0.17.10
```

### Additional context

If a `public` folder also exists in the root directory, it works fine.

![CleanShot project](https://github.com/wxt-dev/wxt/assets/20431351/282c82d6-8efc-48a1-97f6-e86c409ed656)
![CleanShot dev](https://github.com/wxt-dev/wxt/assets/20431351/15a5d756-a1e4-4cb2-98c1-3c9c988db200)

**Comment by aklinker1** [comment]
I had noticed this before, but never tried to figure out what exactly was the cause... So it's moving the public directory out of the root folder... Good find. I think I know how to fix it

**Comment by aklinker1** [comment]
Released fix in [`v0.17.11`](https://github.com/wxt-dev/wxt/releases/tag/v0.17.11)
